README.md

```diff
@@ -81,7 +81,27 @@ The same auto-injection behavior is available for the [Common Custom User Data G
 
 The higher in TeamCity's project hierarchy the configuration parameters are applied, the more widely they apply since the configuration parameters are passed on to all child projects. Child projects can override the configuration parameters, though, and even disable the auto-injection by setting the configuration parameters to empty values.
 
-### Gradle Builds
+### Creating a Gradle Enterprise Connection
+
+A Gradle Enterprise connection can be applied to a project hierarchy on the Connections page of the root project's configuration. In the Add Connection dropdown, select the Gradle Enterprise connection type.
+
+Fill out the Add Connection dialog with the URL for the Gradle Enterprise instance, any plugin or extension versions to be applied, and any other fields in the dialog as needed. Some values, such as the plugin and extension versions, will be pre-populated.
+
+_Note: For Gradle, the Common Custom User Data Gradle plugin must be at least version 1.7 or newer._
+
+_Note: For Maven, the Gradle Enterprise Maven extension and the Common Custom User Data Maven extension are currently hard-coded to versions 1.14.4 and 1.10.1, respectively._
+
+#### Example Configuration
+
+<img width="591" alt="gradle-enterprise-connection-dialog" src="https://user-images.githubusercontent.com/625514/185425268-7d4b363f-941b-49c8-9d88-1e68e2551e9a.png">
```

Comment:
In this screenshot, I purposefully matched the version numbers and url to the screenshots below.